thetower/dtower/tourney_results/constants.py

```diff
@@ -245,4 +245,49 @@ class Options(BaseModel):
     59: ("Wave", "5%", "Crit"),
     60: ("Pizza", "2%", "Damage"),
     61: ("Illuminati", "5%", "Abs def"),
+    62: ("Prismatic Shard", "5%", "DPM"),
+    63: ("Refraction Array", "2%", "Coins"),
+    64: ("Cobweb", "2%", "Health"),
+    65: ("The Fly", "5%", "Abs def"),
+    66: ("Clip Ons", "2%", "Crit"),
+    67: ("Code Stream", "5%", "Damage"),
+    68: ("Summit Starlight", "", ""),  # unreleased
```

Comment:
This is a public repo and worse yet there's no way to censor PRs on github once they have been created so we kinda leaked that :see_no_evil: